lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -1055,7 +1069,7 @@ data PostTransactionFeeOldData (n :: NetworkDiscriminant) = PostTransactionFeeOl
 type ApiBase64 = ApiBytesT 'Base64 ByteString
 
 newtype ApiSerialisedTransaction = ApiSerialisedTransaction
-    { transaction :: ApiT SealedTx
+    { transaction :: (ApiT SealedTx, ApiSealedTxEncoding)
     } deriving stock (Eq, Generic, Show)
       deriving anyclass (NFData)
```

Comment:
```
data ApiSerialisedTransaction = ApiSerialisedTransaction
    { serialisedTxSealed :: ApiT SealedTx
    , serialisedTxEncoding :: ApiSealedTxEncoding
    } deriving stock (Eq, Generic, Show)
      deriving anyclass (NFData)
```
I find this representation more readable at call sites because of the names of the accessor functions, as opposed to meaningless `fst` / `snd`, example:
this line is less readable:
```haskell
let bytes = serialisedTx $ getApiT $ fst $ (tx ^. #transaction)
```
than this one:
```haskell
let bytes = serialisedTx $ getApiT $ serialisedTxSealed $ tx ^. #transaction

Reply:
sounds reasonable - done